src/FileConventions/Library.fs

```diff
@@ -415,10 +415,10 @@ let DoesNamespaceInclude (fileInfo: FileInfo) (word: string) =
 
     if fileText.Any() then
         let rightNamespace =
-            fileText
-            |> Seq.tryFind(fun x -> x.Contains "namespace" && x.Contains word)
+            fileText |> Seq.find(fun x -> x.Contains "namespace")
 
-        rightNamespace <> None
+        rightNamespace.Split().[1].Equals(word)
```

Comment:
extra paren

Reply:
not sure if using 1 here is good practice. gonna ask @aarani's opinion on this one (we had a similar conversation on another PR)

Reply:
Could you please give us an example?

Reply:
- What does no argument to Split function do?
- Just put 1 inside a well-named variable we can't do much except that.

Reply:
@Mersho ping? why there's no answer to this Afshin's question?

Reply:
As a result of our discussion with @parhamsaremi @aarani , this is the most effective solution